packages/expo/src/logs/RemoteLogging.ts

```diff
@@ -112,6 +113,7 @@ async function _sendNextLogBatchAsync(batch: LogEntry[], logUrl: string): Promis
     Accept: 'application/json',
     'Device-Id': await getInstallationIdAsync(),
     'Session-Id': _sessionId,
+    'Device-Platform': Platform.OS,
```

Comment:
Actually noticing now that we use `expo-platform` in a few places as a query param: https://github.com/expo/expo-cli/blob/4a1d73ac49db75233a3372028f78f41df783fbee/packages/xdl/src/start/ExpoUpdatesManifestHandler.ts#L25 would it maybe make sense to use that param just for parity?

Reply:
If we think the purpose of the existing expo-platform header and the one you're adding here are going to be the same indefinitely, I would use the same header. If the headers happen to share the same values right now but their purposes may diverge in the future, I would use a different header.